contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -664,33 +670,33 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
     /**
      * @dev Marks the receipt of a message from the given destination chain ID with the given message ID.
      *
-     * It is possible that the receipt was already received for this message, in which case we return early.
-     * If existing message is found and not yet delivered, we delete it from state and increment the fee/reward
+     * If the receipt was already previously received for this message, returns early.
```

Comment:
nit: sentence may read better if we flip it around: "Function returns early if the receipt..."